TTPChallenge/QuizVC.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  QuizPartOneVC.swift
+//  TTPChallenge
+//
+//  Created by Luna An on 2/14/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class QuizVC: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    @IBOutlet weak var progressLabel: UILabel!
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var tableView: UITableView!
+    
+    var objectsArray: [QuizGroup] = []
+    var allQuestions: [[Question]] = []
+    var respondedQuestions = 0
+    
+    var qa = [String]()
+    var it = [String]()
+    var wd = [String]()
+    var md = [String]()
+    var da = [String]()
+    
+    let techJobs = [
+        "Quality Assurance Analyst",
+        "IT Engineer",
+        "Web Developer",
+        "Mobile Developer",
+        "Data Analyst"
+    ]
+    
+    let personalityData = [
+        
+        //QA
+        ["Enjoy trying out new products",
+         "Stay up to date on the latest and greatest in tech and devices",
+         "Want to get involved in tech, but not sure where to start",
+         "Enjoy puzzles and challenges"],
+        //IT
+        ["Enjoy tinkering with computers and devices",
+         "Are curious about how tech products work",
+         "Enjoy fast-paced work",
+         "Enjoy solving problems"],
+        //WD
+        ["Have a creative spirit",
+         "Love following the latest web trends & technologies",
+         "Think logically and critically",
+         "Have tried or are interested in learning how to code"],
+        //MD
+        ["Are iPhone or Android obsessed",
+         "Are Always on the hunt for the latest cool app",
+         "Have strong preferences for specific technology or apps",
+         "Have tried or areinterested in learning how to code"],
+        //DA
+        ["Able to work across teams and with a variety of stakeholders",
+         "Communicate effectively and build strong relationships with team members",
+         "Have a knack for math, facts and figures",
+         "Love simplifying complex ideas"]
+    ]
```

Comment:
Make these constants. Add them to the Environment.swift file as static const's. See the file for examples. Let me know if you have a question about this